.changeset/tame-foxes-grab.md

```diff
@@ -0,0 +1,28 @@
+---
+"@salt-ds/theme": minor
+---
+
+Deprecated the following tokens:
+
+```diff
+- --salt-status-info-foreground-disabled
+- --salt-status-success-foreground-disabled
+- --salt-status-warning-foreground-disabled
+- --salt-status-error-foreground-disabled
+- --salt-status-static-foreground-disabled
+- --salt-status-negative-foreground-disabled
+- --salt-status-positive-foreground-disabled
+- --salt-status-info-borderColor-disabled
+- --salt-status-success-borderColor-disabled
+- --salt-status-warning-borderColor-disabled
+- --salt-status-error-borderColor-disabled
+- --salt-palette-error-border-disabled
+- --salt-palette-warning-border-disabled
```

Comment:
Note that `--salt-palette-warning-foreground-disabled` used for the corresponding warning status token didnt exist